project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -207,14 +212,27 @@ function c4m_features_og_group_dashboard_form_alter(&$form, &$form_state, $form_
         $flagged_content_summary = render($flagged_content_summary);
 
         $form['flagged_content'] = array(
-          '#markup'   => '<strong>' . t('Following items will be depromoted:') . '</strong>' . $flagged_content_summary,
+          '#markup'   => '<strong>' . t('Following items will be demoted:') . '</strong>' . $flagged_content_summary,
         );
       }
     }
   }
 }
 
 /**
+ * Submit handler for the flag confirm form.
+ *
+ * Executes unflagging of content flagged by c4m_og_content_highlight flag.
+ */
+function c4m_features_og_group_dashboard_promote_flag_submit(&$form, &$form_state) {
```

Comment:
The name of the function is a contradicted to its logic, please change it.
also elaborate the docs.

This logic is not trivial.